developer-docs-site/docs/tutorials/first-move-module.md

```diff
@@ -9,13 +9,13 @@ import TabItem from '@theme/TabItem';
 
 # Your first Move Module
 
-This tutorial details how to write, compile, test, publish, and interact with Move Modules on the Aptos Blockchain. The steps to doing so:
+This tutorial details how to write, compile, test, publish and interact with Move Modules on the Aptos Blockchain. The steps to doing this are:
 
-* Write, compile, and test the Move Module
-* Publish the Move Module to the Aptos Blockchain
-* Initialize and interact with resources of the Move Module
+1. Write, compile and test the Move Module
+2. Publish the Move Module to the Aptos Blockchain
+3. Initialize and interact with resources of the Move Module
 
-This tutorial builds on "[Your first transaction](/tutorials/your-first-transaction)" and borrows that code as a library for this example. The following tutorial contains example code that can be downloaded in its entirety below:
+This tutorial builds on [Your first transaction](/tutorials/your-first-transaction) and borrows the code as a library for this example. The following tutorial contains example code that can be downloaded in its entirety, below:
```

Comment:
I don't think the comma after entirety belongs?

Reply:
Yeah, probably doesn't matter that much. I've removed it 😄